projects/storefrontapp/src/environments/environment.ts

```diff
@@ -8,12 +8,12 @@ import { Environment } from './models/environment.model';
 export const environment: Environment = {
   production: false,
   occBaseUrl:
-    'https://spartacus-dev0.eastus.cloudapp.azure.com:9002' ||
-    // 'https://spartacus-dev3.eastus.cloudapp.azure.com:9002' ||
+    // 'https://spartacus-dev0.eastus.cloudapp.azure.com:9002' ||
+    'https://spartacus-dev3.eastus.cloudapp.azure.com:9002' ||
```

Comment:
I guess we should not merge changes from this file to develop right now.

Reply:
That will be changed right before the my-company to `develop` merge once we switch completely to dev0